ObserverDesignPattern/Observable/StocksObservable.java

```diff
@@ -0,0 +1,6 @@
+public interface StocksObservable {
+    void add(NotificationObserver observer);
+    void remove(NotificationObserver observer);
+    void notifyUsers();
+    void setStockQuantity(int newStocksAdded);
```

Comment:
Do we need it in Subject Interface. 
![observer_uml](https://github.com/FarhanAhmadISM/DesignPatterns/assets/8518507/e645595d-1b58-4852-aab2-bf8e362e75b3)